src/extension/wps/src/main/java/au/org/emii/wps/GoGoDuckProcess.java

```diff
@@ -55,46 +57,54 @@
     public GoGoDuckProcess(WPSResourceManager resourceManager, GeoServerImpl geoServer, GeoServerResourceLoader resourceLoader) {
```

Comment:
The `geoServer` parameter is now unused, can get remove it from the method signature?